tests/tests/models/test_tacotron2.py

```diff
@@ -29,3 +34,129 @@ def test_tacotron2_model(self):
         X, y = model.parse_batch(batch)
         forward_output = model(X)
         assert len(forward_output) == 4
+
+    def test_stft_seed(self, sample_inference_spectrogram, lj_speech_tacotron2):
+
+        torch.random.manual_seed(1234)
+        np.random.seed(1234)
+        nreps = 1
+        text = "I, Sam, am a very good boy."
+        lines = [t for t in text.split("\n") if t]
+        sequences, input_lengths = prepare_input_sequence(
+            lines,
+            cpu_run=True,
+            arpabet=True,
+            symbol_set="nvidia_taco2",
+            text_cleaner=["english_cleaners"],
+        )
+        sequences = sequences.repeat(nreps, 1)
+        speaker_ids = None
+        input_lengths = input_lengths.repeat(1, nreps).squeeze(0)
+        input_ = sequences, input_lengths, speaker_ids, None
+        (
+            mel_outputs,
+            mel_outputs_postnet,
+            gate_outputs,
+            alignments,
+            mel_lengths,
+        ) = lj_speech_tacotron2.inference(input_)
+
+        estimated_vector = rearrange(
+            mel_outputs_postnet.detach().numpy(), "b m t -> (b m t)"
+        )
+        target_vector = rearrange(
+            sample_inference_spectrogram.detach().numpy(), "m t -> (m t)"
+        )
+        rho = np.corrcoef(
+            estimated_vector,
+            target_vector,
+        )[0, 1]
+        assert rho > 0.99
+
+    def test_tf_inference(
+        self,
+        lj_speech_tacotron2,
+        sample_inference_spectrogram,
+        sample_inference_tf_spectrogram,
+    ):
+
+        torch.random.manual_seed(1234)
+        np.random.seed(1234)
+        tf_index = 111
```

Comment:
This is a hard coded value which was determined empirically and then used in the first generation? How about we add a comment explaining it?

Reply:
I wouldn't say it was "determined" but yeah there needs to be a comment about where the target inferences come from.

Reply:
oh you mean the 111 was determined - yes I see